library/kotlin/test/io/envoyproxy/envoymobile/GRPCRequestHeadersBuilderTest.kt

```diff
@@ -0,0 +1,69 @@
+package io.envoyproxy.envoymobile
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+
+class GRPCRequestHeadersBuilderTest {
```

Comment:
would highly recommend using kluent:
https://github.com/MarkusAmshove/Kluent

Reply:
Might be good for future reference (not sure if @buildbreaker has thoughts, I'm not familiar with it 🤷)

Reply:
Certainly open to new frameworks which would help. Would you like to file an issue for us to have a conversation about this @johnpaulmckean?

Reply:
done.

Reply:
Thank you @johnpaulmckean!

Reply:
Thanks! https://github.com/lyft/envoy-mobile/issues/930 for future reference